engines/bastion_katello/app/assets/javascripts/bastion_katello/files/details/views/file-content-views.html

```diff
@@ -24,7 +24,7 @@ <h3 translate>Content Views that contain this File</h3>
       <tbody>
         <tr bst-table-row ng-repeat="version in table.rows">
           <td bst-table-cell>
-            <a ui-sref="content-view.info({contentViewId: version.content_view.id})">
+            <a href="/content_views/{{version.content_view.id}}">
```

Comment:
We just need a statement here that'll link to the products page if the content view ID is the default org one.

Reply:
![image](https://github.com/Katello/katello/assets/14796566/576ed2d7-c12e-4f5e-9c19-9693ef8d31e6)

Reply:
Oh..We should probably not be showing Default organization CV at all..Let's just hide it..Thoughts?

Reply:
Updated to return only non_default CV versions.